Makefile

```diff
@@ -63,22 +81,16 @@ prere:
 	$(PREFIX)cd libft && make re
 
 clean:
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
-	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ)
+	$(PREFIX)rm -f $(OBJS) $(ENDPOINT_OBJ) $(TANIA_ENDPOINT_OBJ) $(VANIA_ENDPOINT_OBJ)  $(TEST_TOOL_OBJS)
```

Comment:
Please remove vania endpoint, we don't have it anymore

Reply:
Also where's testobjs?

Reply:
tell me please what testclean, testfclean should do? i can include testfclean in fclean then.
right now clean and fclean delete whole build folder

Reply:
Basically clean should delete object files, fclean should clean and also delete the executable. I'd suppose in our case clean should delete the folders too.

testclean and testfclean in my design do exactly same but for tests. Since tests won't work without main project's executable, automatic testfclean on fclean makes sense. Whether you want to add testclean on clean - I leave it to you for now, I don't have a strong opinion right away.

Test object files shoudl be deleted in testclean, that's for sure

Reply:
i made mandatory part how you said, in fclean (where i had choice) i simply added `$(PREFIX)rm -f $(TEST_FNAME) $(TEST_TOOL_FNAME)` that testfclean wouldn be called twice.

Reply:
Nope, I told about choice in clean, not fclean :D